src/main/resources/com/puppycrawl/tools/checkstyle/grammars/javadoc/JavadocParser.g4

```diff
@@ -904,11 +908,13 @@ javadocInlineTag:
             CODE_LITERAL (WS | NEWLINE | LEADING_ASTERISK | text)*
             | DOC_ROOT_LITERAL (WS | NEWLINE | LEADING_ASTERISK)*
             | INHERIT_DOC_LITERAL (WS | NEWLINE | LEADING_ASTERISK)*
-            | LINK_LITERAL (WS | NEWLINE | LEADING_ASTERISK)* reference description?
-            | LINKPLAIN_LITERAL (WS | NEWLINE | LEADING_ASTERISK)* reference description?
+            | LINK_LITERAL (WS | NEWLINE | LEADING_ASTERISK)* reference
+            (WS | NEWLINE)* ((WS | NEWLINE)+ description)?
+            | LINKPLAIN_LITERAL (WS | NEWLINE | LEADING_ASTERISK)* reference
+            (WS | NEWLINE)* ((WS | NEWLINE)+ description)?
```

Comment:
@Vladlis , as we discussed in private call:
We need to postpone this changes for later to avoid changes for PACKAGE in this PR.
create separate issue for support of spaces between reference and description and lets see regression in AST more clear.
@ps-sp , please catch Vlad online to receive details.